README.md

```diff
@@ -88,54 +105,26 @@ More examples
 
 TIP: Cursive - teach it to resolve `w/fn` as `clojure.core/fn`.
 
-NOTE: The `wolframite.wolfram` (`w`) ns has vars for all Wolfram symbols at the time of the last release. Check `w/*wolfram-kernel-name*` for kernel type/version and run `(wolframite.impl.wolfram-syms.write-ns/write-ns!)`
+NOTE: The `wolframite.wolfram` (`w`) namespace has vars for all Wolfram symbols at the time of the last release of Wolframite. Check `w/*wolfram-kernel-name*` for kernel type/version and run `(wolframite.impl.wolfram-syms.write-ns/write-ns!)`
 to generate your own wolfram ns with whatever additional symbols your Wolfram/Mathematice has, and/or with custom "aliases". 
 
-#### Learning Wolframite
-
-[Visit our documentation site](https://scicloj.github.io/wolframite/) to learn all you might want to know about using Wolframite.
-
-#### Customizing Wolframite
```

Comment:
moved into Advanced topics. IMO too much detail for the first meeting with Wolframite.